azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -60,4 +57,3 @@ def load_team_arguments(self, _):
         context.argument('devops_organization', options_list=('--organization', '--org'))
         context.argument('detect', **enum_choice_list(_ON_OFF_SWITCH_VALUES))
         context.argument('project', options_list=('--project', '-p'))
-        context.argument('team_id', options_list=('--team-id', '--id'), type=str)
```

Comment:
I think just adding '--id' here would be enough.
`context.argument('team_id', options_list=('--id'), type=str)`

This way we don't have to change the signature of the commands where just team_id is used. 
Also, it works for user because team_id, project _id is abstracted. And it would allign with API naming while writing the commands.

Reply:
Also pylint won't throw any error in this case.

Reply:
As discussed offline, current changes makes more sense. Also reduces the overhead when adding a new command. Hence resolving this